assets/scss/components/_toggle-button.scss

```diff
@@ -2,28 +2,59 @@
 Toggle Button
 
 Markup:
-<span class="toggle-btn {{modifier_class}}">Toggle Button</span>
-
-.toggle-btn--on - The toggle button is on
-.toggle-btn--off - The toggle button is off
+<span class="toggle">
```

Comment:
Gotcha, sorry. It's the wrapper that could do with being more descriptive then... `toggle-buttons
`